static/js/client.js

```diff
@@ -284,6 +290,52 @@ socket.on("add", function(message) {
     updateStatus();
 });
 
+//TODO: UPDATE MODAL STUFFS
+socket.on("fixq", function(message) {
+    if (disable_updates) return;
+    checkAndUpdateSeq(message.seq);
+
+    $("#queue li").each(function(index, item) {
+        if ($(item).data("entryId") == message.id) {
+            $(item).find("button").addClass("hide");
+
+            if ($(item).hasClass("me")) {
+                $(item).find(".remove-button").removeClass("hide");
+                $(item).find(".helping-text").text("Please update your question");
+                $(item).find(".update-question-button").removeClass("hide");
+
+                try {
+                    if (("Notification" in window) && (Notification.permission == "granted")) {
+                        var notification = new Notification("Your question is not detailed enough!", {
+                            "body": "Please update your question."
+                        });
+                    }
+                } catch (error) {
+                    console.log("There was an error showing a browser notification.");
+                }
+            }
+        }
+    });
+});
+
+socket.on("update", function(message) {
+    if (disable_updates) return;
+    checkAndUpdateSeq(message.seq);
+    
+    $("#queue li").each(function(index, item) {
+        if ($(item).data("entryId") == message.id) {
+            $(item).find("button").addClass("hide");
```

Comment:
helping-text isn't cleared once student updates their question. Need to also clear helping-text here

Reply:
fixed!